src/openapi_server/apis/status_api.py

```diff
@@ -38,13 +40,11 @@
 async def healthcheck_get() -> Dict[str, str]:
     """A test endpoint that checks Marklogic is present"""
     client = client_for_basic_auth(HTTPBasicCredentials(username="", password=""))
-    try:
-        client.user_can_view_unpublished_judgments("")
-    except MarklogicUnauthorizedError:
-        pass
-    except MarklogicAPIError as e:
-        return Response(e.default_message, status_code=e.status_code)
-
+    with error_handling():
+        try:
+            client.user_can_view_unpublished_judgments("")
+        except MarklogicUnauthorizedError:  # expected error
```

Comment:
Just out of interest, does anyone know if Marklogic allows for healthchecks without having to catch unauthorized requests?

Reply:
seems like there might be something: https://docs.marklogic.com/11.0/guide/release-notes/en/new-features-in-marklogic-11/enhanced-health-check-monitoring.html
I'll have more of a look tomorrow.

not for this PR obviously

Reply:
Interesting and good to know -- could make the healthcheck significanly more informative!